frontend/src/pages/BYONImages/utils.ts

```diff
@@ -26,4 +26,4 @@ export const getEnabledStatus = (image: BYONImage): number =>
     : ImageEnabledStatus.DISABLED;
 
 export const filterBlankPackages = (packages: BYONImagePackage[]) =>
-  packages.filter((p) => p.name.trim() || p.version.trim());
+  packages.filter((p) => p.name.trim() !== '' && p.version.trim() !== '');
```

Comment:
I think once we had a discussion on the UX meeting, only when these 2 fields are both empty we filter it out. The admin should know what they are inputting (like a package without version).

Reply:
oh gotcha sorry. ill change it

Reply:
ok fixed it